helm/cilium/values.yaml

```diff
@@ -1603,12 +1669,13 @@ prometheus:
         replacement: ${1}
     # -- Metrics relabeling configs for the ServiceMonitor cilium-agent
     metricRelabelings: ~
+    # -- Set to `true` and helm will not check for monitoring.coreos.com/v1 CRDs before deploying
+    trustCRDsExist: false
   # -- Metrics that should be enabled or disabled from the default metric
   # list. (+metric_foo to enable metric_foo , -metric_bar to disable
   # metric_bar).
-  # ref: https://docs.cilium.io/en/stable/operations/metrics/#exported-metrics
-  metrics: 
-   - +cilium_bpf_map_pressure
+  # ref: https://docs.cilium.io/en/stable/observability/metrics/
+  metrics: ~
```

Comment:
@njuettner @paurosello could this be a mistake? We specifically introduced this in https://github.com/giantswarm/cilium-app/commit/a321566c834a1efa211e2bcabf5a58a1c4e5b442.

Reply:
yes, we need that back in